### PR TITLE
Add architecture information to the UI

### PR DESCRIPTION
**Pull request by joaquimrocha**
This PR adds the new architecture information of packages and channels to the UI.

Screenshot from 2019-12-20 22-15-50
Screenshot from 2019-12-20 22-15-29
Screenshot from 2019-12-20 22-15-21
Screenshot from 2019-12-20 22-03-41

### CODE REVIEW

File: frontend/src/js/constants/helpers.js [hunk 1]
```diff
@@ -5,6 +5,13 @@ import orange from '@material-ui/core/colors/orange';
 import red from '@material-ui/core/colors/red';
 import moment from 'moment';
 
+export const ARCHES = {
```
Comment (krnowak) [inline frontend/src/js/constants/helpers.js]: Maybe a comment that it matches (or needs to match) values in `pkg/api/arches.go`?

**Comment by krnowak** [comment]
Maybe in a future iteration we could add some colored labels for arches, so it will be easier to quickly distinguish between ARM64 and AMD64 channels/packages.

**Comment by joaquimrocha** [comment]
> Maybe in a future iteration we could add some colored labels for arches, so it will be easier to quickly distinguish between ARM64 and AMD64 channels/packages.

Sure. That section needs a UX review anyway.

File: frontend/src/js/constants/helpers.js (hunk 1 above)
Comment (joaquimrocha) [reply frontend/src/js/constants/helpers.js]: I will do that.

**Comment by joaquimrocha** [comment]
Merging after having added the comment @krnowak suggested.

**Pull request merged**